curriculum/challenges/english/25-front-end-development/workshop-loan-qualification-checker/66c8ba975ee7230e29f6c4b5.md

```diff
@@ -0,0 +1,61 @@
+---
+id: 66c8ba975ee7230e29f6c4b5
+title: Step 14
+challengeType: 1
+dashedName: step-14
+---
+
+# --description--
+
+Change the `loanMessage` inside the first `else if` block to `"- Car loan\n- Condo loan\n"` and remove the console statement.
```

Comment:
I feel like steps 14 and 15 should be combined

Reply:
I will look into this when I start writing the tests